packages/flutter/lib/src/widgets/interactive_viewer.dart

```diff
@@ -85,6 +87,12 @@ class InteractiveViewer extends StatefulWidget {
            && boundaryMargin.left.isFinite)),
        super(key: key);
 
+  /// If true, panning is only allowed in the direction of the main axes.
+  ///
+  /// In other words, when this is true, diagonal panning is not allowed. This
+  /// is a common pattern in tables where data is displayed in columns and rows.
```

Comment:
It appears to me that if the user initiates a horizontal drag, then they won't be able to change the direction to vertical later (in the same gesture). Maybe add documentation for this behavior too?

Reply:
That's right.  I'll document it.  It seems to be how Google Sheets works on my iPhone too.